browser/css/device-mobile.css

```diff
@@ -517,6 +517,25 @@ nav:not(.spreadsheet-color-indicator) ~ #toolbar-wrapper #toolbar-up.w2ui-toolba
 #toolbar-down table.w2ui-button .w2ui-tb-image {
 	margin: 5px 9px !important;
 }
+#toolbar-down table.w2ui-button .textcolor {
+	margin: 5px 9px 0px !important;
+}
+#toolbar-down table.w2ui-button .backcolor {
+	margin: 5px 9px 0px !important;
```

Comment:
maybe better to use `margin-bottom` instead, so to avoid duplications since we can inherit the rest